app/components/api-utils.js

```diff
@@ -29,9 +29,12 @@ const asyncFetch = (url, fetchOptions, parseBody) => {
                 if (!endsWith(url, 'health')) {
                     logger.info(`Status: ${res.status}`);
                 }
+                logger.info(`res.ok: ${res.ok}`);
```

Comment:
@shepda added this line, unsure if needed

Reply:
I don't think so because we have looking in the conditional for the case if `res.ok` is set and we also have logging if `res.ok` is not set and we've got an error